tests/foreman/cli/test_activationkey.py

```diff
@@ -799,8 +799,8 @@ def test_positive_add_redhat_and_custom_products(module_org):
     assert {REPOSET['rhst7'], repo['name']} == {pc['name'] for pc in content}
 
 
-@pytest.mark.stubbed
-def test_positive_delete_manifest(module_org):
+@pytest.mark.tier2
+def test_positive_delete_manifest(module_manifest_org):
```

Comment:
I'm concerned that deleting the manifest from the subscription for this module scoped manifest provided by a fixture will negatively impact other test functions using this module scoped dependency.

Have you been running this test case in isolation, or along with other test cases that use this same module scoped fixture?

Reply:
Yes, @mshriver you're right It will impact other tests, I ran this test in isolation. I'll use function scoped fixture, think that will be better.